modules/apps/bean-portlet/bean-portlet-registration-impl/src/main/java/com/liferay/bean/portlet/registration/internal/BeanPortletRegistrarImpl.java

```diff
@@ -408,7 +283,7 @@ public void step4ApplicationScopedInitializedAsync(
 			HttpWhiteboardConstants.HTTP_WHITEBOARD_SERVLET_PATTERN,
 			"/portlet-servlet/*");
 
-		_serviceRegistrations.add(
+		serviceRegistrations.add(
 			bundleContext.registerService(
 				Servlet.class,
 				new PortletServlet() {
```

Comment:
@ngriffin7a is there a reason for this anonymous sub-class of PortletServlet? why not just use PortletServlet directly?

Reply:
@shuyangzhou I suppose either would work. Feel free to give `PortletServlet.class` a try if you think that makes the code read better.